argus/config_generator/windows/cb_init.py

```diff
@@ -93,7 +94,9 @@ def _get_service(self, service_type):
             This can be HTTP, ConfigDrive, EC2, OpenNebula,
             CloudStack or MAAS.
         """
-        return '.'.join([util.SERVICES_PREFIX, self.SERVICES[service_type]])
+        if self.SERVICES[service_type]:
```

Comment:
if self.SERVICES.get would be better as it will not error out if the key does not exist